### PR TITLE
Assembly guide

### PR DESCRIPTION
**Pull request by sweetlilmre**
Hi,

I put together an assembly guide (based on mistakes I made when building one for the first time), I hope you find this useful.
Please let me know if you'd like any edits / additions.
-(e)

### CODE REVIEW

**Review by scrapcomputing — APPROVED** [review]
This is awesome ! Thanks for sharing detailed assembly instructions and photos !

By the way, have you noticed any noise coming from the Pi/DAC ? Someone raised this issue with the original WavetablePi (https://github.com/scrapcomputing/WavetablePi/issues/4) and I am wondering if this is a more widespread issue.

File: assembly.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Assembling the Horizontal WavetablePi
```
Comment (scrapcomputing) [inline assembly.md:1]: Feel free to add your handle or name here like: `# Assembling the Horizontal WavetablePi (by sweetlilmre)`

**Comment by sweetlilmre** [comment]
I haven't noticed any DAC noise, but I've only tested very briefly. I'll give it another go and see if I pick anything up.

File: assembly.md (hunk 1 above)
Comment (sweetlilmre) [reply assembly.md:1]: Something strange is happening, I added a commit and commented on this issue, but neither are pulling through

Comment (sweetlilmre) [reply assembly.md:1]: Ah.. weird. It was a GH sync issue. I  seem to have fixed it but editing the PR and changing the base (well not really, just pointing it to your main branch again).

**Pull request merged**